tox.ini

```diff
@@ -1,2 +1,16 @@
 [flake8]
 max-line-length=88
+exclude=.cache,.venv,.git
+docstring-convention=all
+import-order-style=pycharm
+ignore=
+    # Todo comments
+    T000
+    # Docstring Whitespace
+    D203,D212,D214,D215,
+    # Docstring Quotes
+    D301,D302,
+    # Docstring Content
+    D400,D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417,
```

Comment:
The only plugin installed for `flake8` at the moment is `flake8-annotations`. These error codes come from plugins that are not installed for this project.